src/external-repos.md

```diff
@@ -29,6 +27,20 @@ to these tools should be filed against the tools directly in their respective
 upstream repositories. The exception is that when rustc changes are required to
 implement a new tool feature or test, that should happen in one collective rustc PR.
 
+`subtree` dependencies are currently managed by two distinct approaches:
+
+* Using `git subtree`
```

Comment:
portable-simd also uses this; it seems to be entirely missing from this list?

Reply:
The list was incomplete before, I added a few entries, but since there was no global list before that, I didn't notice portable-simd. Good catch! Added it in https://github.com/rust-lang/rustc-dev-guide/pull/2214.